README.md

```diff
@@ -48,14 +48,15 @@ Now start up the container, by mounting the local git clone directory as
 `/ceph`:
 
     # docker run -itd \
-      -v $PWD:/ceph \
-      -v ~/.ccache:/root/.ccache \
+      -v <pathToCephRepo>:/ceph \
+      -v $PWD/scripts:/scripts \
+      -v <pathToCCacheDir>:/root/.ccache \
```

Comment:
If you are replacing `~/.ccache`, you should adapt the documentation (see line 63)